Tests/MapboxNavigationTests/RouteControllerSnapshotTests.swift

```diff
@@ -21,7 +18,7 @@ class RouteControllerSnapshotTests: FBSnapshotTestCase {
     }
     
     func testRouteSnappingOvershooting() {
-        let options = NavigationMatchOptions(coordinates: [
+        let coordinates:[Waypoint] = [
```

Comment:
This is an array of waypoints, not an array of coordinates, so the variable name is inaccurate. But `NavigationMatchOptions(coordinates:profileIdentifier:)` takes an array of coordinates.

Reply:
Thanks for the suggestion, I'll update this part.